assembly.html.markdown

```diff
@@ -0,0 +1,218 @@
+---
+language: assembly
+filename: learn.asm
+contributors:
+    - ["", ""]
+---
+
+Assembly language is a low-level programming language for a computer
+
+```asm
+; Comments
+
+section .data
+; it contains data, constants
+
+; (variable-name) define-directive initial-value
+byte_ db 0 ; 1 byte
+word_ dw 0 ; 2 bytes
+doubleword_ dw 0 ; 4 bytes
```

Comment:
`s/dw/dd/`

In general, this section is pretty confusing. Could you add a little more explanation about what each part of the line means, what `db`, `dw`, `dd` mean, and that it's OK to specify multiple values, and they all get turned into an array. Also why `()` around variable name.

Side note, I don't see a `.rodata` section, but I think it's worth mentioning.